api-docs/rst/dev-guide/api-operations/methods/get-get-scaling-group-state-v1.0-tenantid-groups-groupid-state.rst

```diff
@@ -14,14 +14,17 @@ This operation retrieves the current state of a scaling group.
 The *GroupState* object consists of the following properties:
 
 
-
-*  *paused*. Specifies whether execution of scaling policies for the group is currently suspended. If this value is set to true, the group will not scale up or down. All policy execution calls are ignored while this value is set to true.
+*  *paused*. Specifies whether execution of scaling policies for the group is currently suspended. If this value
+   is set to TRUE, the group will not scale up or down. All scheduled and API-generated policy executions are suspended, and convergence will
+   not be triggered while this value is set to TRUE. Any *POST* calls to the :ref:`/converge <post-create-scaling-group-v1.1-tenantid-converge>` operation and any *POST* calls to the :ref:`/execute <post-execute-policy-v1.0-tenantid-groups-groupid-policies-policyid-execute>`
+   operation will error with a `403GroupPausedError` error message if convergence is paused. If this value is set FALSE, all scaling and convergence activities, as well as all policy execution calls are resumed.
```

Comment:
Reposting revised version of the paused parameter value descriptions:

If paused=TRUE, the group does not scale up or down. All scheduled or API-generated policy operations are suspended, and convergence cannot be triggered. When the group is paused, any POST requests for :ref:/converge <post-create-scaling-group-v1.1-tenantid-converge> or :ref:/execute <post-execute-policy-v1.0-tenantid-groups-groupid-policies-policyid-execute> operations return a 403 GroupPausedError response.

If paused=FALSE, all group scaling and convergence operations resume and scheduled or API-generated policy exectuions are allowed.